src/otx/algorithms/common/adapters/mmcv/utils/automatic_bs.py

```diff
@@ -127,6 +127,10 @@ def _set_batch_size(cfg, batch_size: int):
     else:
         cfg.data.train_dataloader["samples_per_gpu"] = batch_size
 
+        for custom_hook in cfg.custom_hooks:
+            if custom_hook["type"] == "AdaptiveRepeatDataHook":
+                custom_hook["train_batch_size"] = batch_size
+
```

Comment:
Why don't you use `update_or_add_custom_hook()`?

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2510/commits/e1cf24fe04c36af2963f098052c248475a68e2b7